tools/wptserve/wptserve/response.py

```diff
@@ -154,6 +159,8 @@ def maybe_set(key, value):
 
     def unset_cookie(self, name):
         """Remove a cookie from those that are being sent with the response"""
+        if PY3:
+            name = name.decode('iso-8859-1')
```

Comment:
How do we know this is a type that can be decoded? Should we be using `isomorphic_decode`?

Reply:
Well, we expect this API to only accept bytes, which will be enforced once we require all string literals to be prefixed, but yeah I've switched to `isomorphic_decode` which can accept text strings, too.